java/src/main/java/io/cucumber/messages/Convertor.java

```diff
@@ -11,7 +11,7 @@ private Convertor(){
     }
 
     public static Exception toMessage(Throwable t) {
-        return new Exception(t.getClass().getName(), t.getMessage());
+        return new Exception(t.getClass().getName(), t.getMessage(), null);
```

Comment:
@mpkorstanje not sure if you'd want to change anything here at this point e.g. do a `printStackTrace` - happy to include it if you'd like.